R/upload_model.R

```diff
@@ -205,8 +205,14 @@ deploy.pbpk <- function(user.input, out.vars, create.params, create.inits,
 
   # Ask the user for a a model title
   title <- readline("Title of the model: ")
+  if (nchar(title) < 3){
+    stop("Please provide a title that is more than 3 characters long")
+  }
   # Ask the user for a short model description
   description  <- readline("Short description of the model:")
+  if (nchar(description) < 3){
+    stop("Please provide a description that is more than 3 characters long")
+  }
```

Comment:
this is not needed for the description, but ok you can leave it so some description is added